docs/source/v1.3.md.inc

```diff
@@ -12,6 +12,7 @@
 
 [//]: # (- Whatever (#000 by @whoever))
 
-[//]: # (### :bug: Bug fixes)
+### :bug: Bug fixes)
 
-[//]: # (- Whatever (#000 by @whoever))
+- Fix pandas 2.0 compatibility (#732 by @larsoner)
```

Comment:
I'd add this to the code health section instead as

```
- Support report generation with pandas 2.0 (#732 by @larsoner)
```
WDYT? Since it's not a bug

Reply:
To me it was a bug that we were knowingly using a deprecated API, and users will see it as an error (no pipeline can even run with latest pandas). So to me it's above the level of "code health" which sounds like stuff maintainers should mostly care about